### ISSUE TITLE
There is no mention of 'Data Down, Actions up' in the guides

### ISSUE THREAD

**Issue by theroncross** [emberjs/guides#1787]
1. Is this the adopted philosophy at this point?

2. Does it need to be explicitly 'taught'?

**Comment by locks** [comment]
> 1. Is this the adopted philosophy at this point?

It is a very useful frame with which to approach structuring a front-end application, but the fact that curly components are still two-way bound, there isn't really a way to enforce the one-way binding aspect of DDAU without introducing a lot of overhead and subtle inconsistencies.

> 2. Does it need to be explicitly 'taught'?

Well, it would help if there was a consensus on what DDAU even is :P

**Comment by theroncross** [comment]
> It is a very useful frame with which to approach structuring a front-end application, but the fact that curly components are still two-way bound, there isn't really a way to enforce the one-way binding aspect of DDAU without introducing a lot of overhead and subtle inconsistencies.

Right, but don't we want to encourage new Ember-ists to adopt this style of architecture?

> Well, it would help if there was a consensus on what DDAU even is :P

I was more just thinking of explicitly putting the 'DDAU' name on what's already in the guides.

**Comment by aripollak** [comment]
It seems like angle-bracket components are supposed to enforce DDAU, but they don't seem to be mentioned in the guides at all, and there are a lot of references to two-way binding. Would it make sense for the tutorial to talk about the new-style components & input tags instead of {{component}} and {{input}}?

**Comment by locks** [comment]
@aripollak angle-bracket components don't exist yet, so everything is two-way bound by default.
I'd say that `{{input}}` is still valuable, especially at the leaf node of a data flow.
